docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-destinations/snowflake/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Setup Snowflake Destination"
-sidebar_position: 80
+title: "Snowflake"
+sidebar_position: 30
```

Comment:
Could you move it to between BigQuery and GCS?

Reply:
Sure!